File: client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/operations/NotEqualsOperation.java
package org.evomaster.client.java.controller.mongo.operations;

/**
 * Represent $ne operation.
 * Selects the documents where the value of the field is not equal to the specified value.
 * This includes documents that do not contain the field.
 */
public class NotEqualsOperation<V> extends ComparisonOperation<V>{
    public NotEqualsOperation(String fieldName, V value) {
        super(fieldName, value);
    }
}

File: client-java/controller/src/main/java/org/evomaster/client/java/controller/mongo/operations/NotEqualsOperation.kt
package org.evomaster.client.java.controller.mongo.operations

/**
 * Represent $ne operation.
 * Selects the documents where the value of the field is not equal to the specified value.
 * This includes documents that do not contain the field.
 */
class NotEqualsOperation<V>(
    override val fieldName: String,
    override val value: V
): ComparisonOperation<V>(fieldName, value)